plugins/jobs/server/models/job.py

```diff
@@ -89,6 +89,19 @@ def list(self, user, types=None, statuses=None,
                                                 limit=limit, offset=offset):
             yield r
 
+    def listAll(self, limit=0, offset=0, sort=None, currentUser=None):
+        """
+        List all jobs.
+
+        :param limit: The page limit.
+        :param offset: The page offset
+        :param sort: The sort field.
+        :param currentUser: User for access filtering.
+        :deprecated: use list method instead
```

Comment:
[This](https://github.com/girder/girder/blob/e34549d653c63c7fc4a5ae49b5f9b773974e8122/girder/api/v1/file.py#L204) is the only other instance of deprecation I could find in our Python. 
 I don't care which style we use, but I'd like to be consistent.

    Multipart uploads are @deprecated as of v2.2.0.